lib/msf/base/sessions/mettle_config.rb

```diff
@@ -71,6 +71,19 @@ def generate_config(opts={})
           opts[:name] ||= ds['PayloadProcessCommandLine']
         end
 
+        if ds['RemoteMeterpreterDebugFile'] != ''
+          opts[:log_file] ||= ds['RemoteMeterpreterDebugFile']
+        end
+
+        if ds['MeterpreterDebugLevel'].is_a? Integer
```

Comment:
Not entirely sure why this is required, as I used `OptInt`, and it still comes over as a string from venom

Reply:
Datastore serialization is special. This is shouldn't be needed though, because all `Integer`s in Ruby will happily return themselves when asked to `#to_i`.

Reply:
Also predictable results with nil and empty string.... good call.